pkg/kubelet/kubelet_test.go

```diff
@@ -2719,26 +2719,59 @@ func TestUpdateNewNodeOutOfDiskStatusWithTransitionFrequency(t *testing.T) {
 	}
 }
 
-// FIXME: Enable me..
-func testDockerRuntimeVersion(t *testing.T) {
+func TestDockerRuntimeVersion(t *testing.T) {
 	testKubelet := newTestKubelet(t)
 	kubelet := testKubelet.kubelet
 	fakeRuntime := testKubelet.fakeRuntime
 	fakeRuntime.RuntimeType = "docker"
-	fakeRuntime.VersionInfo = "1.5.0"
-	fakeRuntime.APIVersionInfo = "1.18"
+	fakeRuntime.VersionInfo = "1.10.0-rc1-fc24"
+	fakeRuntime.APIVersionInfo = "1.22"
 	kubeClient := testKubelet.fakeKubeClient
 	kubeClient.ReactionChain = testclient.NewSimpleFake(&api.NodeList{Items: []api.Node{
-		{ObjectMeta: api.ObjectMeta{Name: testKubeletHostname}},
+		{
```

Comment:
I wanted to add a test case to cover this, and in lieu of adding a new testing exactly like this one, I re-enabled this test and updated it. Some of its mocks were outdated and incomplete, but didn't want to solve the original issue without ensuring it is covered in a test.

The mocks are updated from other tests I had to touch in the previous PR. So a bit copypasta, but is also consistent.